osc-ui/src/main/java/org/osc/core/broker/view/vc/securitygroup/BindSecurityGroupWindow.java

```diff
@@ -418,9 +413,10 @@ private void serviceTableClicked(long itemId) {
 		}
 	}
 
-	private boolean isMultiplePoliciesSupported(Long vsId) {
+	private boolean isBindedWithMultiplePolicies(Long vsId) {
 		for (VirtualSystemPolicyBindingDto binding : this.allBindings) {
-			if (binding.getVirtualSystemId().equals(vsId) && binding.isMultiplePoliciesSupported()) {
+			if (binding.getVirtualSystemId().equals(vsId) && binding.isMultiplePoliciesSupported() && binding.isBinded()
```

Comment:
can we extract the condition below into another method and use it in line 152 as well?

`binding.isMultiplePoliciesSupported() && binding.isBinded() && binding.getPolicyIds().size() > 1`

we can name the overloaded method the same and add javadoc to both methods.
`isBindedWithMultiplePolicies(VirtualSystemPolicyBindingDto binding)`